app/pages/cif/project-revision/[projectRevision]/index.tsx

```diff
@@ -171,18 +168,13 @@ export function ProjectRevision({
             />
           </Grid.Col>
           <Grid.Col>
-            <ProjecManagerForm
-              ref={projectManagerFormRef}
-              formData={
-                query.projectRevision.projectManagerFormChange.newFormData
-              }
-              onChange={(change) =>
-                handleChange(
-                  query.projectRevision.projectManagerFormChange,
-                  change.formData
-                )
+            <ProjecManagerFormGroup
+              query={query}
+              revision={query.projectRevision}
+              projectManagerFormRef={projectManagerFormRef}
+              setValidatingForm={(validator) =>
+                (projectManagerFormRef.current = validator)
```

Comment:
I'd really like to find better wording for this function / callback if you have ideas, I'm quite unhappy with what I chose when I first implemented it.
This is just confusing